src/OpenLoco/src/Jukebox.cpp

```diff
@@ -0,0 +1,148 @@
+#include "Jukebox.h"
+#include "Audio/Audio.h"
+#include "Config.h"
+#include "Date.h"
+#include "Environment.h"
+#include "Localisation/StringIds.h"
+#include <OpenLoco/Interop/Interop.hpp>
+#include <numeric>
+
+using namespace OpenLoco::Environment;
+using namespace OpenLoco::Interop;
```

Comment:
Interop can be removed once the two `loco_global`s have been integrated, i.e.:
```suggestion
#include <numeric>

using namespace OpenLoco::Environment;
```